.circleci/config.yml

```diff
@@ -7,12 +7,16 @@ workflows:
     jobs:
       - architect/push-to-app-catalog:
           context: "architect"
-          # executor: "app-build-suite" # uncomment this if you want automatic metadata generation
-          name: "package and push {APP-NAME} chart"
-          app_catalog: "giantswarm-playground-catalog"
-          app_catalog_test: "giantswarm-playground-test-catalog"
-          chart: "{APP-NAME}"
+          executor: "app-build-suite"
+          name: "push-to-default-app-catalog"
+          app_catalog: "giantswarm-catalog"
```

Comment:
Shouldn't it be `default-catalog` ?

Reply:
that's what i get for not paying attention